.gitignore

```diff
@@ -1,6 +1,23 @@
 reports/
 logs/
 
+# Distribution / packaging
```

Comment:
I fixed this issue please give me my money

Reply:
This PR seems old with a lot of conflicts. BTW, this is IDP, not IBP